pkg/arvo/lib/quiz.hoon

```diff
@@ -0,0 +1,38 @@
+|_  [eny=@uv runs=@ud]
+++  check  |=  [vax=vase]
+  =.  runs  ?:(=(0 runs) 100 runs)
+  =+  size=1
+  =+  sax=(slot 6 vax)
+  =+  run-i=0
+  =+  rng=~(. og eny)
+  |-
+  =+  sam=(~(fill quiz [size rng]) sax)
+  =+  res=(slam vax sam)
+  ^-  ?
```

Comment:
Typical to put this on the same line as the barhep.

Reply:
That's recarding the kethep, github doesn't display the line i'm referring to very clearly.